src/DynamoCoreWpf/UI/Themes/Modern/Ports.xaml

```diff
@@ -109,7 +109,7 @@
                     </Rectangle.Margin>
                 </Rectangle>
                 <Rectangle Name="highlightOverlay"
-                       Fill="{Binding Path=ShouldKeepListStructure, Converter={StaticResource KeepListStructureHighlighColorConverter}}"
+                       Fill="{Binding Path=ShouldKeepListStructure, Converter={StaticResource KeepListStructureHighlightColorConverter}}"
```

Comment:
@mjkkirschner Is the Binding Path still `ShouldKeepListStructure`?

Reply:
yes, as this is on the portViewModel and not the model